.github/helper/api_test.py

```
@@ -0,0 +1,179 @@
+        'args': ['project', 'function_signature']
+    },
+    '/api/easy-params-far-reach': {
+        'args': ['project']
+    },
+    '/api/far-reach-low-cov-fuzz-keyword': {
+        'args': ['project']
+    },
+    '/api/project-repository': {
+        'args': ['project']
+    },
+    '/api/project-repository': {
```

Ruff (0.8.2)

Dictionary key literal `'/api/project-repository'` repeated

Remove repeated key literal `'/api/project-repository'`

(F601)